tests/inventory_adoption.yaml

```diff
@@ -0,0 +1,5 @@
+[local]
```

Comment:
We should drop this file i think?

Reply:
that would drop from the finished version yes